src/openforms/submissions/tests/test_submission_attachment.py

```diff
@@ -0,0 +1,347 @@
+import os
+
+from django.core.files import File
+from django.test import TestCase, override_settings
+
+from PIL import Image, UnidentifiedImageError
+from privates.test import temp_private_root
+
+from openforms.forms.tests.factories import FormStepFactory
+from openforms.submissions.attachments import (
+    append_file_num_postfix,
+    attach_uploads_to_submission_step,
+    clean_mime_type,
+    cleanup_submission_temporary_uploaded_files,
+    resize_attachment,
+    resolve_uploads_from_data,
+)
+from openforms.submissions.models import SubmissionFileAttachment
+from openforms.submissions.tests.factories import (
+    SubmissionFileAttachmentFactory,
+    SubmissionStepFactory,
+    TemporaryFileUploadFactory,
+)
+
+
+@temp_private_root()
+class SubmissionAttachmentTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.test_image_path = os.path.join(
+            os.path.dirname(__file__), "files", "image-256x256.png"
+        )
+
+    # TODO private storage
```

Comment:
what's this TODO about?

Reply:
Something that has been fixed, so I removed the todo.